docs/changes.txt

```diff
@@ -74,6 +74,9 @@ INCOMPATIBLE CHANGES SINCE 3.1.0:
 
 - The enum value wxTASKBAR_JUMP_LIST_DESTIONATION, which was added in 3.1.0,
   contains a typo and has been renamed to wxTASKBAR_JUMP_LIST_DESTINATION.
+  
+- wxZipOutputStream will now automatically convert filenames to UTF-8, if no
```

Comment:
I'd say "... will now use UTF-8 to encode filenames and comments in generated archive instead of the current locale encoding by default".